index.d.ts

```diff
@@ -339,4 +343,41 @@ declare module 'binance-api-node' {
         isOrderWorking: boolean;
         isBuyerMaker: boolean;
     }
+
+    interface MyTrade {
+        id: number;
+        orderId: number;
+        price: string;
+        qty: string;
+        commission: string;
+        commissionAsset: string;
+        time: number;
+        isBuyer: boolean;
+        isMaker: boolean;
+        isBestMatch: boolean;
+    }
+
+    interface QueryOrderResult {
+        symbol: string;
+        orderId: number;
+        clientOrderId: string;
+        price: string;
+        origQty: string;
+        executedQty: string;
+        status: string;
+        timeInForce: string;
+        type: string;
+        side: string;
+        stopPrice: string;
+        icebergQty: string;
+        time: number;
+        isWorking: boolean;
+    }
+
+    interface CancelOrderResult {

```

Comment:
Is that a `Result`? To me it looks like a `Request` because it is created when calling `cancelOrder`.

The Java API also names it `CancelOrderRequest`:
- https://github.com/binance-exchange/binance-java-api/blob/master/src/main/java/com/binance/api/client/domain/account/request/CancelOrderRequest.java

Reply:
Yes, it is a `Result`.